scipy/stats/_stats_py.py

```diff
@@ -100,16 +101,16 @@
            'expectile']
 
 
-def _chk_asarray(a, axis):
+def _chk_asarray(a, axis, xp=np):
```

Comment:
I think I would prefer `_chk_asarray(a, axis, *, xp=None)` with
```python
if xp is None:
	xp = array_namespace(a)
```
but let me know if you disagree.

Reply:
My plan was to always pass `xp` when updating functions to use the array API. For any function that doesn't pass `xp` when it calls `_chk_asarray`, `np` is correct, so it seems unnecessary to call `array_namespace` to determine that.

Reply:
Alright. I have heard others object to having `np` as a default value though - a pattern used in cluster is `xp=None`, with `xp = np if xp is None else xp`,

Reply:
I went ahead and changed it.